src/xdsl/printer.py

```diff
@@ -312,11 +325,10 @@ def _print_op_attributes(self, attributes: Dict[str, Attribute]) -> None:
         self.print("[")
 
         attribute_list = [p for p in attributes.items()]
-        self.print("\"%s\" = " % attribute_list[0][0])
-        self.print_attribute(attribute_list[0][1])
+        self._print_attr_string(attribute_list[0][0], attribute_list[0][1],
+                                True)
```

Comment:
If anyone sees a way to make this nicer, please tell me!

Reply:
There should be a `print_list` method in `Printer` that should do what you want ;)
That way, you can remove your `if not first: self.print(", ")`

Reply:
Wow, that is way better!